examples/all-clusters-app/esp32/main/main.cpp

```diff
@@ -158,10 +160,18 @@ class EditAttributeListModel : public ListScreen::Model
         int n;
         if (sscanf(value.c_str(), "%d", &n) == 1)
         {
+            auto & attribute = std::get<1>(std::get<1>(std::get<1>(devices[d])[e])[c])[a];
+            auto & name      = std::get<0>(attribute);
+
             ESP_LOGI(TAG, "editing attribute as integer: %d (%s)", n, i == 0 ? "+" : "-");
             n += (i == 0) ? 1 : -1;
             char buffer[32];
             sprintf(buffer, "%d", n);
+            if (!strncmp(name.c_str(), "Temperature", sizeof(name.c_str())))
```

Comment:
sizeof won't do the right thing here.  If `name` is an `std::string` here, why not just `if (name == "Temperature")`?